src/index.js

```diff
@@ -3,6 +3,7 @@ const fs = require("fs");
 
 const { authenticate } = require("./mtm");
 const { registerService } = require("./discovery-api-client");
+const { validateInputs } = require("./validations");
```

Comment:
I can't write this comment on the correct line, as this is not part of this PR, but why do we do this:
let dryRun = core.getInput("dry-run");
dryRun = !(dryRun === "false");

Reply:
it's funny, the input from yaml is not parsed correctly by github core package. it comes as string. any non-empty string is a truly value in JS/TS so we added this logic